### PR TITLE
delete parent; make plugin versions upgradable in children repo; add java 14 and 15 support.

### PR DESCRIPTION
**Pull request by XenoAmess**
Hi. I found this repo kindof useful, but plugin versions are locked, which is a drawback point.
So please review this pr.
Thanks.
Also, I deleted the oss-parent, as it is actually deprecated. See their github page for more details.

### CODE REVIEW

**Comment by XenoAmess** [comment]
@russgold any ideas?

**Comment by russgold** [comment]
Whoops - just received notification of these changes. I'll take a look, thanks.

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Make this your parent POM:
 
 ```
     <parent>
-        <groupId>com.meterware</groupId>
+        <groupId>com.xenoamess</groupId>
```
Comment (russgold) [inline README.md:9]: Are you branching or looking to update the original?

File: pom.xml [hunk 2]
```diff
@@ -337,6 +502,27 @@
...
+                            <toolchains>
+                                <jdk>
+                                    <version>${base.java.version}</version>
+                                    <vendor>${toolchain.vendor}</vendor>
+                                </jdk>
+                            </toolchains>
+                        </configuration>
+                    </plugin>
 
                     <!-- enable java9 - java13 compilations -->
```
Comment (russgold) [inline pom.xml:527]: Should probably update this line

**Review by russgold — CHANGES_REQUESTED** [review]
The changes look good - but I cannot accept the PR with a change to the group name. Can you revert that in your PR?

**Comment by XenoAmess** [comment]
> The changes look good - but I cannot accept the PR with a change to the group name. Can you revert that in your PR?

I changed it only because I waited for your response for too long, and I need to use that,so I decide to change that,thus I can push it into maven central.
If you be agree to accept this pr I'm glad to rename it back, and rebase it.

File: pom.xml (hunk 2 above)
Comment (XenoAmess) [reply pom.xml]: @russgold done

File: README.md (hunk 1 above)
Comment (XenoAmess) [reply README.md]: @russgold done

**Comment by XenoAmess** [comment]
@russgold done. please do a review. thanks!

**Review by russgold — APPROVED** [review]

**Pull request merged**